server.py

```diff
@@ -30,7 +35,6 @@ def load_user(user_id):
 # ROUTES
 # Home page
 @app.route('/', methods=['GET'])
-@app.route('/home', methods=['GET'])
 def home_page():
```

Comment:
Can we rename this `index`? It shouldn't have any redirects at this point, but this makes more sense since there is no home page.